### DISCUSSION TITLE
Scaling Hubs Cloud for a large (10k+) conference

### DISCUSSION THREAD

**Discussion by wchest** [mozilla/hubs#2852] [Hubs Cloud Questions] [answered]
Hi all, we're getting ready to host a conference with up to 15,000 participants and would like to use our Hubs Cloud environment to host some networking events. I figure a very small percent of that total will be on Hubs at any given time, so was planning to have something like 30 rooms or so (understanding the 25 person best practice per room). Do you have advice on how many servers we should spin up in our cluster to support this? We have two c5 instances at the moment and thinking that's probably not going to be sufficient.

**Comment by robinkwilson** [comment, marked as answer]
From my calculations, 2 app and 2 streaming c5.2xlarge servers would accommodate 15,000 participants. You can scale the 2x2 servers up or down during the live event as needed via [Updating the Stack](https://hubs.mozilla.com/docs/hubs-cloud-aws-updating-the-stack.html).

I'd also review the [Minimizing cost documentation](https://hubs.mozilla.com/docs/hubs-cloud-aws-costs.html#minimize-your-costs---a-user-story) for a flow for minimizing costs setting up and time leading up to your event.

**Reply by wchest** [reply]
Amazing, thank you for the response @robin-k-wilson . Really appreciate it.

**Comment by davegoopot** [comment]
@wchest , you might need to keep an eye on the capabilities of the devices connecting to the room too. Anecdotally we've seen problems with rendering and especially sound on mobile phones when hitting 10-15 participants.
